### PR TITLE
Adding tier-0 rbd test suite for nautilus build

### PR DESCRIPTION
**Pull request by manasagowri**
Signed-off-by: mgowri <mgowri@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

**Comment by psathyan** [comment]
The code formatting check has failed. Please run `black` against your changes

File: tests/rbd/test_rbd.py [hunk 1]
```diff
@@ -75,13 +73,27 @@ def run(ceph_cluster, **kwargs) -> int:
...
+    if "5." in rhbuild:
+        nodes[0].exec_command("ceph config set mon mon_allow_pool_delete true")
+        nodes[0].exec_command("ceph orch restart mon")
+    else:
+        nodes[0].exec_command(sudo=True, cmd="ceph config set mon mon_allow_pool_delete true")
+        mon_nodes = ceph_cluster.get_nodes(role="mon")
+        for ceph_mon in mon_nodes:
+            ceph_mon.exec_command(
+                sudo=True,
+                cmd="systemctl restart ceph-mon@{}".format(ceph_mon.hostname),
```
Comment (psathyan) [inline tests/rbd/test_rbd.py:93]: Opt to use `cmd=f"systemctl restart ceph-mon@{ceph_mon.hostname}"`

**Review by psathyan — COMMENTED** [review]
Nice work on reducing the number of required nodes. A minor improvement but the rest looks good.

File: tests/rbd/test_rbd.py (hunk 1 above)
Comment (manasagowri) [reply tests/rbd/test_rbd.py]: Done. Made the necessary changes.

**Review by psathyan — APPROVED** [review]

**Comment by manasagowri** [comment]
> Nice work on reducing the number of required nodes. A minor improvement but the rest looks good.

Thank you :)

**Pull request merged**